### PR TITLE
Adds TSFeatures class

### PR DESCRIPTION
**Pull request by harishkashyap**
- Adds progressive drift

### CODE REVIEW

File: tsfeatures/TSFeatures.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            (str): 'PD' if progressive drift. 'NA' if not.
+
+        '''
+        delta = self.bus_current_train_df.loc[sample_no + 1,
+                                              'BUS CURRENT'] - self.bus_current_train_df.loc[sample_no,
+                                                                                             'BUS CURRENT']
+        if delta < 0.01:
+            return 'PD'
+        else:
+            return 'NAN'
```
Comment (sridharrajaram) [inline tsfeatures/TSFeatures.py:37]: this should be "NA"

**Review by sridharrajaram — CHANGES_REQUESTED** [review]
its good overall. need little change

File: tsfeatures/TSFeatures.py (hunk 1 above)
Comment (Apoorvagang) [reply tsfeatures/TSFeatures.py:37]: This need to be changed.

**Review by Apoorvagang — CHANGES_REQUESTED** [review]
a little bit change is required.

**Review by sridharrajaram — APPROVED** [review]
Looks good

**Review by Apoorvagang — APPROVED** [review]

**Pull request merged**